src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/product/form/convert-to-simple-product.ts

```diff
@@ -0,0 +1,91 @@
+import {EventsHash} from 'backbone';
+import * as _ from 'underscore';
+
+import BaseView = require('pimui/js/view/base');
+const Dialog = require('pim/dialog');
+const __ = require('oro/translator');
+const messenger = require('oro/messenger');
+const Routing = require('routing');
+const router = require('pim/router');
+
+const template = require('pim/template/product/convert-to-simple-product');
+
+interface Config {
+    url: string;
+}
+
+class ConvertToSimpleProduct extends BaseView {
+    private readonly template = _.template(template);
+
+    private readonly config: Config;
+
+    constructor(options: { config: Config }) {
+        super({...options, ...{className: 'AknDropdown-menuLink', tagName: 'button'}});
+
+        this.config = {...this.config, ...options.config};
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public events(): EventsHash {
+        return {
+            'click': this.convert,
+        };
+    }
+
+    public render(): BaseView {
+        const formData = this.getFormData();
+        if ('product' === formData.meta.model_type && null !== formData.parent && this.isAuthorized()) {
+            this.$el.html(this.template({
```

Comment:
Maybe we can just have 

```suggestion
            this.$el.html( __('pim_enrich.entity.product.module.convert_variant_to_simple.label'));
```

and remove the html template?